eclide/RepositoryFrame.h

```diff
@@ -60,8 +60,6 @@ class CRepositoryFrameT :	public CRepositorySlotImpl,
     {
         T * pT = static_cast<T*>(this);
         ContextState state;
-        if (s.mods.size() + s.attrs.size() == 0)
-            return;
```

Comment:
@GordonSmith I have opened another jira for Default Actions (IDE-648). This pull request is simply to remove the blocking of the popup for now, putting the IDE back to the previous defaults which has not changed.